packages/node-firebird-driver/src/test/tests.ts

```diff
@@ -22,10 +57,12 @@ export function runCommonTests(client: Client) {
 		let tmpDir: string;
 
 		function getTempFile(name: string): string {
-			return `${tmpDir}/${name}`;
+			const p = `${(baseTmpDir || tmpDir)}/${name}`;
```

Comment:
Seems weird relation with (local) `tmpDir` and remote servers.

Reply:
I agree, but running tests using a remote connection requires the database path to pre-exist. In your initial implementation, temp directories are generated locally and doesn't serve the remote scenario.

Reply:
If the problem is about the term `baseTmpDir`, I can rename it to remoteDbDir if you prefer. Or do you have a better name for this?

Reply:
But seems you are still creating local temporary directory even with remote host. You could specially treat automatic temporary directories for embedded, localhost and 127.0.0.1 and do not create it for remote.